src/app/qgisapp.cpp

```diff
@@ -8409,7 +8409,9 @@ void QgisApp::deleteSelected( QgsMapLayer *layer, QWidget *, bool checkFeaturesV
                                       Qgis::MessageLevel::Info );
     return;
   }
-  //display a warning
+
+  //display an "ooutside of view" warning
```

Comment:
Typo: "ooutside" >:)

Reply:
Huh, not sure why it says "reviewed". I added the above as a "Single Comment". I am totally not fit to properly review code here!